cmd/rac/cli_test.go

```diff
@@ -77,7 +77,7 @@ func Test_processFiles(t *testing.T) {
 
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			dir, err := ioutil.TempDir("/tmp", "mats-testing")
+			dir, err := ioutil.TempDir("", "mats-testing*")
```

Comment:
what's that asterisk?

Reply:
adds a random number to the file (in the case of concurrent tests)

Reply:
But I'm quite sure TempDir and TempFile did that without an asterisk. I have a feeling I logged the filenames and saw random numbers in the end.

Reply:
yes - correct the asterisk is redundant at the end of the filname/dirname.